### PR TITLE
Simplify user journey for the SH installation

### PR DESCRIPTION
**Pull request by pchico83**
The idea of this PR is to simplify the user journey for SH installation and having a clear link to share for SH installations

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -10,12 +10,11 @@ module.exports = {
     {
       "Get started": [
       "overview",
-      "get-started/preparation",
       {
         "type": "category",
-        "label": "Install Okteto",
+        "label": "Install Okteto SH",
```
Comment (rberrelleza) [inline sidebars.js:15]: I don't think we should put 'SH', it can lead to confusion (what is Okteto SH?). I think "Okteto" is fine, it's what we use across the docs and other content when talking about the platform.  And I think the guide is clear enough that if you are using SaaS, you can skip to the next step.

File: src/content/get-started/overview.mdx [hunk 2]
```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
 
 In order to fully install Okteto, you'll need the following:
 
-- A domain to which you can add a wildcard DNS record
 - A Kubernetes cluster
+- An Okteto License
+- A domain to which you can add a wildcard DNS record
```
Comment (rberrelleza) [inline src/content/get-started/overview.mdx:21]: I've always found this wording confusing.  You don't add records to a domain. Maybe we just leave it as: "a domain and the ability to create DNS records for it"?

File: src/content/get-started/overview.mdx [hunk 3]
```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
...
 - A pool with at least 3 nodes with a minimum of 4CPUs and 16 GB of Memory
 - 250 GB per disk
 
 You'll be using the cluster's API server endpoint when configuring Okteto.
 
 > Our installation guides assume Okteto will be running in a dedicated cluster. We recommend [contacting our team](https://www.okteto.com/contact/) if you plan on installing Okteto in a cluster with other workloads.
 
+### License
+
+You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
```
Comment (rberrelleza) [inline src/content/get-started/overview.mdx:41]: You also receive a license on a trial. We should reword this to prevent confusion.  We have wording in other pages on this.

File: src/content/get-started/overview.mdx [hunk 4]
```diff
@@ -51,12 +57,46 @@ Be sure to configure your kubectl to talk to your newly minted cluster.
...
+Okteto components:
 
-## Deploying Okteto
+1. NGINX Ingress
+1. Okteto/API
+1. Okteto/Buildkit
+1. Okteto/Frontend
+1. Okteto/Migration
+1. Okteto/Registry
+1. Okteto/Validation Webhook
```
Comment (rberrelleza) [inline src/content/get-started/overview.mdx:72]: I don't think that we need to speficy the okteto-level components at this level. This is the first time the prospect sees the helm installation, this level of detail is not necessary. I'd prefer to have a separate page where we explain the components, architecture, etc.

File: src/content/get-started/overview.mdx [hunk 5]
```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
 
 In order to fully install Okteto, you'll need the following:
```
Comment (rberrelleza) [inline src/content/get-started/overview.mdx:17]: this comment is for the full guide: I think this guide is trying to do too much. Our assumption is that this is a getting started, meant for those who are just learning about okteto and want to try it out. I think there's a lot of redundancy between this doc, the quickstart guides of each cloud provider, and the rest of the docs. 

I think that on this guide we should remove pretty much everything, and just stay with the following:
1. What is Okteto (one paragraph max)
2. How to get a license (start your trial here!)
3. Video on how to install 
4. Link to the quickstart install guides  (the list of cloud-specific providers)
5. IF you want to learn more about how configure and troubleshoot Okteto Self-Hosted go here (with a link to the more in-depth doc section)
For a beginner, we don't want them to install everything from zero. we want them to pick their cloud provider, copy the commands, follow the guide, and have a working instance in less than 1 hour. Anything that doesn't contribute to that (explain the components, troubleshooting , how to uninstall) I think should live somewhere else in our docs.

File: src/content/get-started/overview.mdx (hunk 3 above)
Comment (pchico83) [reply src/content/get-started/overview.mdx:41]: @rberrelleza could you share the right wording here?

File: src/content/get-started/overview.mdx (hunk 5 above)
Comment (pchico83) [reply src/content/get-started/overview.mdx]: I made another pass, @rberrelleza could you give it another pass?
I have seen other installation guides, and it's quite common to mention troublleshooting an uninstall in the installation overview doc

File: src/content/get-started/cloud-provider-guides/gke.mdx [hunk 6]
```diff
@@ -137,4 +130,4 @@ After a successful installation, you can access your Okteto instance at `https:/
 
 > **Important**: The default installation is not recommended for production use. We highly advise following the steps outlined in the [Admin Dashboard](admin/dashboard.mdx#installation) before giving your team access to your Okteto instance.
 
-The default installation is all that's required to follow our [getting started](get-started/install-okteto-cli.mdx) guide.
+[Next](get-started/install-okteto-cli.mdx) we're going to install and configure the Okteto CLI with your Okteto instance.
```
Comment (ifbyol) [inline src/content/get-started/cloud-provider-guides/gke.mdx:133]: I'm not sure how easy to find those links are. They are the last sentence of the screen and `Next` or `getting started` are short words to see them easily. I think links are good to take the user to other relevant sections, but if we want to have a flow/guide of different steps, links in a paragraph are hard to find.

Should we consider to add a flow with X steps with the typical footer to next or previous section? In my mind, it would be something like:
* `get-started/overview` would have links to the public cloud providers guides.
* Each public provider guide would have a previous button to `get-started/overview` and a next button to "install the CLI"
* "Install CLI" would have a previous step to "choose the public cloud provider" and a next button to "deploy your first dev environment".
* "Deploy your first dev environment" would have a previous link to "install the CLI" and a next page to "What's next?" where we can have an index to other sections in the docs like: complete your installation (bring your own certificate, configure the auth provider, etc..), features discoverability (preview envs, install GitHub integration, catalog, private registries, etc...)

**Review by ifbyol — APPROVED** [review]
In general it looks good, I added a couple of comments as suggestions but nothing blocker

File: src/content/get-started/cloud-provider-guides/gke.mdx (hunk 6 above)
Comment (codyjlandstrom) [reply src/content/get-started/cloud-provider-guides/gke.mdx:133]: I agree with Nacho here. I think the hit area for this link should be larger and very obvious as the next step. Maybe even a button using `pagination_next` ?

File: src/content/get-started/overview.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- [Install Okteto in Google Kubernetes Engine](get-started/cloud-provider-guides/gke.mdx)
+- [Install Okteto in DigitalOcean Kubernetes](get-started/cloud-provider-guides/digitalocean.mdx)
+
+## Video Tutorials
+
+If you need help installing Okteto on your Kubernetes cluster, we have a playlist of videos guiding you through each of the installation steps. Check them out here:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLVcXGaCKg-Y0XGujg_G7vCIGd_JEAQ4Sq" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Troubleshooting
```
[the remark this answers is not in the document]
Comment (codyjlandstrom) [reply src/content/get-started/overview.mdx:41]: Could we change the "Troubleshooting" header to be "Community and Support" and add the link in there?

File: src/content/get-started/overview.mdx (hunk 3 above)
Comment (rberrelleza) [reply src/content/get-started/overview.mdx]: I think what we have on https://deploy-preview-596--okteto-docs.netlify.app/docs/1.17/self-hosted/manage/okteto-license/#how-to-obtain-a-license is a good fit for this block.

File: src/content/get-started/overview.mdx (hunk 7 above)
[the remark this answers is not in the document]
Comment (rberrelleza) [reply src/content/get-started/overview.mdx:41]: I agree with @codyjlandstrom , I like the header we used on https://github.com/okteto/docs/blob/9e01a2b3654d68670dee001051675573897ce14e/src/content/self-hosted/overview.mdx#L60

File: src/content/get-started/overview.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+We've done our best to make Okteto easy to install and operate. [The troubleshooting page](self-hosted/manage/troubleshooting.mdx) has tips and tricks to help troubleshoot common issues.
+
+You can also [reach out to support](https://okteto.com/support) if you need assistance.
+
+## Uninstalling Okteto
```
Comment (rberrelleza) [inline src/content/get-started/overview.mdx:47]: Since we don't show them how to install it on this page, I think we should remove this from here. maybe we can add the uninstall instructions on the troubleshooting section? (or maybe a full section on how to uninstall Okteto?)

File: sidebars.js [hunk 9]
```diff
@@ -120,7 +119,6 @@ module.exports = {
   },
   {
     "Okteto Self-Hosted": [
-      "self-hosted/overview-self-hosted",
```
Comment (rberrelleza) [inline sidebars.js:123]: CAn you remove the overview.mdx since we are not displaying it everywhere, and for now redirect to get-started/overview?

**Review by rberrelleza — APPROVED** [review]
My requests are nice to haves, I think it's a great improvement already1

File: src/content/get-started/cloud-provider-guides/gke.mdx (hunk 6 above)
Comment (pchico83) [reply src/content/get-started/cloud-provider-guides/gke.mdx:133]: +1 I am going to merge as it is a create a dedicated PR for this
